### DISCUSSION TITLE
ajuda!!

### DISCUSSION THREAD

**Discussion by elciod** [balta-io/forum#224] [Tira Dúvidas] [answered]
bom dia, preciso de uma ajuda aula react sem querer excluir o arquivo index.tsx!! diretório-dev- todo ---templete typescript

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @elciod, como vai?

Se não estiver na lixeira, não é possível recuperar, você pode criar outro e seguir o código fonte que tem aqui no GitHub!

**Comment by elciod** [comment]
bom dia certo obrigado


Em seg., 16 de jan. de 2023 às 15:05, Andre Baltieri <
***@***.***> escreveu:
